app/views/entity_categories/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="item-form-container">
+  <%= form_for(@entity_category, url: group_category_entity_categories_path(@category), method: :post, html: { class: "item-form" }) do |f| %>
+    <div class="input-field">
+      <%= f.text_field :name,  placeholder: "Enter name", required: true %>
+    </div>
+
+    <br>
+    
+    <div class="input-field">
+      <%= f.number_field :amount, placeholder: "Enter amount", required: true %>
+    </div>
+
+  <div class="add-btn">
+    <%= f.submit "Create Item" %>
+  </div>
+  <% end %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=categories%20(mandatory%20at%20least%20one)), the form should have 3 input fields: name, amount, and categories. Please add the `categories` input field to this form to add at least one category to your transaction.